dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Content/Site.css

```diff
@@ -85,6 +85,11 @@ label {
     background-color: rgba(0, 130, 114, 0.75);
     border: none;
 }
+.btn-link {
```

Comment:
This was needed to get a button that's styled like a link to line up with other links